library/weno/src/AdminProperties.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * AdminProperties class.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Rx\Weno;
+
+class AdminProperties
+{
+
+    public function __construct()
+    {}
+
+
+
+    public function addNarcotics()
+    {
+
+  /*
+  * Import the narcotics into the table database from SQL file
+  *
+  */
+        $sqlNarc = file_get_contents('../../contrib/weno/narc.sql');
+
```

Comment:
Above the sqlStatementNoLog, do the following:
``````
// Settings to drastically speed up import with InnoDB
sqlStatementNoLog("SET autocommit=0");
sqlStatementNoLog("START TRANSACTION");
``````

Reply:
Done